tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "declaration": true,
+      "strict": true,
+      "moduleResolution": "node",
+      "module": "commonjs",
+      "target": "es5",
```

Comment:
Now that we support node 6 or greater, i think we can safely target es6

Reply:
Done